src/main/java/seedu/address/logic/commands/AddLessonCommand.java

```diff
@@ -26,6 +26,9 @@ public CommandResult execute(Model model) throws CommandException {
 
         if (model.hasLessonClashWith(lesson)) {
             Lesson clashingLesson = model.getLessonClashWith(lesson);
+            if (clashingLesson.getName().equals(lesson.getName())) {
```

Comment:
Is this only for lessons which have the same time? 

I forgot if we decided on unique lesson names (which i think may be classified as a bug - what if a tutor wants to have the same lesson on two diff dates with differnet people?)